### PR TITLE
Add brendandburns to kubernetes-client owners

### PR DESCRIPTION
**Pull request by brendandburns**
This is needed for me to use Azure pipelines for CI/CD for CSharp and other projects.

(I'm also the primary maintainer for most of the client libraries)

Thanks!

### CODE REVIEW

**Comment by justaugustus** [comment]
Hey @brendandburns, org admins are also (by default) maintainers of the GitHub teams that they belong to, which is why the test is failing [here](https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/org/615/pull-org-test-all/400):
```
FAIL: //config:go_default_test (see /root/.cache/bazel/_bazel_root/e50ebc2a7cac8d93968e5988863c67ce/execroot/__main__/bazel-out/k8-fastbuild/testlogs/config/go_default_test/test.log)
INFO: From Testing //config:go_default_test:
==================== Test output for //config:go_default_test:
--- FAIL: TestAllOrgs (0.09s)
config_test.go:248: The team haskell-admins in org Kubernetes Clients has org admins listed as members; these users should be in the maintainers list instead, and cannot be on the members list: brendandburns
config_test.go:248: The team ruby-admins in org Kubernetes Clients has org admins listed as members; these users should be in the maintainers list instead, and cannot be on the members list: brendandburns
config_test.go:248: The team gen-admins in org Kubernetes Clients has org admins listed as members; these users should be in the maintainers list instead, and cannot be on the members list: brendandburns
config_test.go:248: The team javascript-admins in org Kubernetes Clients has org admins listed as members; these users should be in the maintainers list instead, and cannot be on the members list: brendandburns
config_test.go:248: The team csharp-admins in org Kubernetes Clients has org admins listed as members; these users should be in the maintainers list instead, and cannot be on the members list: brendandburns
config_test.go:248: The team java-admins in org Kubernetes Clients has org admins listed as members; these users should be in the maintainers list instead, and cannot be on the members list: brendandburns
FAIL
```

To fix it, you can add a `maintainers` section with your username to each team you need to be a part of, and remove yourself from the `members` field e.g.,
```yaml
  csharp-admins:
    description: admin access to csharp
    maintainers:
    - brendandburns
    members:
    - krabhishek8260
    privacy: closed
```

File: config/kubernetes-client/org.yaml [hunk 1]
```diff
@@ -5,6 +5,7 @@ has_organization_projects: true
 has_repository_projects: true
 members_can_create_repositories: false
 admins:
+- brendandburns
```
Comment (nikhita) [inline config/kubernetes-client/org.yaml]: Only the GitHub admin team can be in the `admins` list: https://github.com/kubernetes/community/tree/master/github-management#github-administration-team

To get admin permissions across repos in the kubernetes-client org, can you create teams for each repo like below? :grimacing: 

```yaml
  csharp-admins:
    description: admin access to csharp
    members:
    - brendandburns
    - krabhishek8260
    privacy: closed
```

I'll then grant admin privileges to these teams.

Comment (rlenferink) [reply config/kubernetes-client/org.yaml]: I took the liberty to go ahead with this and create the `go-admins` and `go-base-admins` teams (#618) which were the only repositories not belonging to a team (yet).

**Comment by nikhita** [comment]
Closing in favor of https://github.com/kubernetes/org/pull/618.

**Pull request closed without merging**